docs/_locale/de/installation.pot

```diff
@@ -673,8 +673,8 @@ msgid "Download the ISO on which you want to base the container. In this example
 msgstr "Download the ISO on which you want to base the container. In this example, the name of the ISO is ``vyos-1.4-rolling-202308240020-amd64.iso``. If you created a custom IPv6-enabled network, the ``docker run`` command below will require that this network be included as the ``--net`` parameter to ``docker run``."
 
 #: ../../installation/virtual/proxmox.rst:37
-msgid "Download the rolling release iso from https://vyos.net/get/nightly-builds/. Non-subscribers can always get the LTS release by building it from source. Instructions can be found in the :ref:`build` section of this manual. VyOS source code repository is available https://github.com/vyos/vyos-build."
-msgstr "Download the rolling release iso from https://vyos.net/get/nightly-builds/. Non-subscribers can always get the LTS release by building it from source. Instructions can be found in the :ref:`build` section of this manual. VyOS source code repository is available https://github.com/vyos/vyos-build."
+msgid "Download the rolling release iso from https://vyos.net/get/nightly-builds/. Non-subscribers are not able to build the LTS but can always build the rolling release from source. Instructions can be found in the :ref:`build` section of this manual. VyOS source code repository is available https://github.com/vyos/vyos-build."
```

Comment:
would you please delete all .pot files from the PR? The translation will be manage with a service and have this in the PR are prevent from backport the PR.

Reply:
Would a `find . -type f -name "*.pot" -delete` just do?

Reply:
Or do you mean to revert all the changes to the pot files?

Reply:
Fixed